app/domain/authentication/util/check_authenticator_secret_exists.rb

```diff
@@ -0,0 +1,40 @@
+require 'command_class'
+require 'conjur/fetch_required_secrets'
+
+module Authentication
+  module Util
+
+    # Command class that returns boolean value indicating if resource exists
+    CheckAuthenticatorSecretExists = CommandClass.new(
+      dependencies: {
+        resource_class: ::Resource,
+        logger: Rails.logger
+      },
+      inputs: %i[conjur_account authenticator_name service_id var_name]
+    ) do
+
+      def call
+        check_authenticator_secret_exists
+      end
+
+      private
+
+      def check_authenticator_secret_exists
+        authenticator_secret_exists
+      end
+
+      def authenticator_secret_exists
+        !resource.nil?
+      end
+
+      def resource
+        @logger.debug(LogMessages::Authentication::AuthnJwt::FetchingJwtConfigurationValue.new(resource_id))
```

Comment:
the class is generic but the log is from JWT

Reply:
Removed